### ISSUE TITLE
Dispatch CAD Feature

### ISSUE THREAD

**Issue by DACAxThor** [Linesmerrill/police-cad#527]
**Describe the solution you'd like**
[_A clear and concise description of what you want to happen._]

-  Like the signal 100 but a 10-3 button that does the same but is in yellow and makes a different noise

**Implementation Details**
[_Add any and all specific things you would like to see implemented. These can include things such as limiting the input for a input box to only include specific characters. (This can be as technical as you would like to make it, including code examples)_]

-

**Comment by SowinskiBraeden** [comment]
Can you elaborate more please? What is the purpose of this button? Is it meant for just police? Etc.

**Comment by DACAxThor** [comment]
10-3 is code word for stop transmitting. When there is a lot of chaos and
the dispatcher cannot get his voice loud he uses the 10-3 button where
every police officer can see it and they can be quiet. Used only by
dispatcher and the active system is on police and dispatchers only.

**Comment by DACAxThor** [comment]
Very important it also comes with a sound because for officers that are not
always looking at the CAD can still notice it.

**Comment by SowinskiBraeden** [comment]
Ok thanks, Ill try to let you know if any developments are made.

**Comment by DACAxThor** [comment]
Thanks

**Comment by Linesmerrill** [comment]
Yeah I don't see a problem with adding this. Let me mock out a solution and see what you guys think. 👍

**Comment by DACAxThor** [comment]
Okay perfect thanks.

**Comment by Linesmerrill** [comment]
@DACAxThor let me know if this is kinda what you were looking for?

Dispatch:
![image](https://user-images.githubusercontent.com/15384983/135357265-ae45126a-6734-43d2-8453-24572bf09315.png)

![image](https://user-images.githubusercontent.com/15384983/135357394-6497c64c-3298-4631-922d-1e56e8582160.png)


-- 

Police:
![image](https://user-images.githubusercontent.com/15384983/135357321-414b15f5-fa34-49c3-9c9b-373ccae76480.png)

**Comment by SowinskiBraeden** [comment]
Looks good👍

**Comment by Linesmerrill** [comment]
Work located here: https://github.com/Linesmerrill/police-cad/pull/528

**Comment by Linesmerrill** [comment]
We just need a good sound clip to play, @DACAxThor do you have any suggestions on sounds we could use for this notification?

**Comment by DACAxThor** [comment]
Looks amazing I will try to get a sound.

**Comment by DACAxThor** [comment]
For the beeps just use the same as what you already just change the tempo.
Make it faster or slower. Also can you change the wording. Is read of
dispatch has the mic can you put 10-3 is active

**Comment by Linesmerrill** [comment]
ok let me add the sound and then yeah ill modify the wording.

**Comment by Linesmerrill** [comment]
Dispatch Page:
![image](https://user-images.githubusercontent.com/15384983/135721722-ce961da3-6fb5-4421-8dc2-0d0c9959873f.png)


---

Police Page:
![image](https://user-images.githubusercontent.com/15384983/135721748-28e9eec1-17c4-4a85-b013-faefb5e1725f.png)

**Comment by DACAxThor** [comment]
Amazing thanks

**Comment by Linesmerrill** [comment]
Cool! once it finish our testing on this it should be released in a couple days 👍 

Thanks again for this amazing suggestion. If there are any other things you would like to see us add or modify, just create a ticket in here and we will address it.

**Comment by DACAxThor** [comment]
Will do and thank you so much.
